### ISSUE TITLE
Update ceterms:sameAs for use with Transfer Value Profile use case

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/Schema-Development#716]
Per our 9-10-2020 meeting:

We should extend the domain and range of [`ceterms:sameAs`](https://credreg.net/ctdl/terms/sameAs) to include:
- ceterms:Credential (and subclasses)
- ceterms:LearningOpportunityProfile
- ceterms:AssessmentProfile

This would enable a blank node (say, for instance, in a Transfer Value Profile's `@graph`) to have its own information (such as competencies) _and_ to reference the "real" data that it is describing.  This enables ACE to assert that a given learning opportunity teaches certain competencies while also referencing the "real" learning opportunity that was published/maintained by its actual owner as a separate, standalone entity.

**Comment by philbarker** [comment]
Couldn't you just give it the same [@]id? There's no need (is there?) that every reference to an existing resource to have all the information about that resource just because it has some.

This would avoid having a BNode -- I think BNodes are useful for abstractions beyond those we cover (e.g. any Bachellor degree from MIT) but otherwise best avoided. 

Also `[ceterms:sameAs](https://credreg.net/ctdl/terms/sameAs#sameAs)` refers to "another entity that unambiguously indicates the identity of the entity being described" you would be using it here to point the same entity as the one being described.

**Comment by siuc-nate** [comment]
In the registry, only the true owner of the data, or an organization authorized to publish on the owner's behalf, may publish authoritative data (that is, standalone record with its own CTID and therefore its own full URI `@id`) about that owner or that owner's property (credentials, etc.).  Since ACE doesn't own the learning opportunities or assessments, it can't publish those standalone authoritative records about them, hence using bnodes for them.  

However, since ACE does possess data about the competencies and whatnot that factor directly into their transfer value assignment, we want to enable them to include that data in their TVP data's bnodes (regardless of whether those same competencies are mentioned in an authoritative learning opportunity/assessment record that may or may not exist).  Hence the need to have a bnode that is about the same "thing" as the authoritative record (if one exists), hence the suggestion of `sameAs`.

In other words, we want to enable ACE to assert all kinds of things about learning opportunities and assessments without breaking our policy around who is allowed to publish what, while also referencing authoritative records where they exist.

**Comment by philbarker** [comment]
@siuc-nate OK, that's a good rationale, thanks.
 
Depending on how pedantic you feel, you might want to tweak the definition of sameAs so that it doesn't require that it point to a different entity. Something like "Another source of information about the entity being described" or similar.

**Comment by siuc-nate** [comment]
Per our 9-14-2020 meeting:

Proposal:

Remove:
> **Subject:** ceterms:sameAs
> **Predicate:** rdfs:comment
> **Object:** "Another entity that unambiguously indicates the identity of the entity being described."

> **Subject:** ceterms:sameAs
> **Predicate:** dct:description
> **Object:** "Entities that may indicate identity include, but are not limited to, descriptions of entities in open databases such as DBpedia and Wikidata or other platforms."

Add:
> **Subject:** ceterms:sameAs
> **Predicate:** rdfs:comment
> **Object:** "Another source of information about the entity being described."

> **Subject:** ceterms:sameAs
> **Predicate:** dct:description
> **Object:** "Other sources may include descriptions of entities in open databases such as DBpedia, Wikidata, the Credential Engine Registry, or other platforms."

> **Subject:** ceterms:sameAs
> **Predicate:** schema:domainIncludes
> **Object:** ceterms:Credential (and subclasses), ceterms:AssessmentProfile, ceterms:LearningOpportunityProfile, ceterms:Occupation, ceterms:Job

> **Subject:** ceterms:sameAs
> **Predicate:** schema:rangeIncludes
> **Object:** ceterms:Credential (and subclasses), ceterms:AssessmentProfile, ceterms:LearningOpportunityProfile, ceterms:Occupation, ceterms:Job

**Comment by siuc-nate** [comment]
These changes have been made in pending CTDL and noted in the history tracking.
